### DISCUSSION TITLE
Can i overwrite the default description for nvim mappings?

### DISCUSSION THREAD

**Discussion by lucasmarinb** [mrjones2014/legendary.nvim#294] [Q&A] [answered]
I've recently installed this plugin and i really like it, but i noticed that the default description of some mappings doesn't match the keywords i remember them with.
For instance i'd like to search for `Alternate files` and get the `<C-^>`, but the current description is `Edit last edited files`.

**Comment by mrjones2014** [comment, marked as answer]
Hm, it’s not currently possible but it should be do-able if you open an issue.

**Reply by lucasmarinb** [reply]
Hey, that's great to know!
Thanks for your answer.
